core/move.js

```diff
@@ -50,3 +51,18 @@ function child(side /*: SideName */, src /*: Metadata */, dst /*: Metadata */) {
   src.childMove = true
   move(side, src, dst)
 }
+
+function convertToDestinationAddition(
+  side /*: SideName */,
+  src /*: Metadata */,
+  dst /*: Metadata */
+) {
+  // Delete source
+  if (src.moveTo) delete src.moveTo
```

Comment:
@nono 	I added the 2 `delete` calls after writing unit tests for this method. If you're ok with those changes, I'll merge the PR.

Reply:
OK for the second delete, but for the first delete, I don't see in which case it happens. Can you enlighten me?

Reply:
When we call `move(side, src, dst)`, we add a `moveTo` attribute on `src` and a `moveFrom` attribute to `dst`.
So I'm removing both to make sure nothing's left from the move.